Neos.Flow/Classes/Http/Helper/RequestInformationHelper.php

```diff
@@ -100,10 +100,14 @@ public static function renderRequestHeaders(RequestInterface $request): string
         $renderedHeaders = '';
         $headers = $request->getHeaders();
         if ($headers instanceof Headers) {
-            $renderedHeaders .= $headers->__toString();
+            $renderedHeaders .= preg_replace('/Authorization:[^\r\n]+/', 'Authorization: ****', $headers->__toString());
```

Comment:
Note: This if will be gone with later versions, hence why I put the preg_replace here only instead of doing it after the if for both cases.